packages/hoppscotch-data/src/environment/v/1.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from "zod"
+import { defineVersion } from "verzod"
+import { V0_SCHEMA } from "./0"
+
+export const V1_SCHEMA = z.object({
+  v: z.literal(1),
+  id: z.string(),
```

Comment:
Is `id` a required field? It's marked optional in the `v0` schema.

Reply:
its optional because in older version there is case where id will not be present, and in newer version its mandatory